flink-kubernetes-operator-api/pom.xml

```diff
@@ -126,6 +126,13 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
```

Comment:
This dependency is now added twice if I see correctly

Reply:
good catch fixed